pkg/acceptance/manualtestprofiles/profiles.go

```diff
@@ -0,0 +1,6 @@
+package manualtestprofiles
+
+const (
```

Comment:
Nit: IMO splitting the profile names to 2 separate packages is an overkill.

Reply:
it was done for a nice import only, nothing more, I wanted to have a clear separation between test and manual profiles

Reply:
I moved them directly to the manual auth testing package for separation without an overkill